internal/provider/helper_test.go

```diff
@@ -277,3 +278,10 @@ func testAccGitlabProjectStart(t *testing.T) testAccGitlabProjectContext {
 		project: project,
 	}
 }
+
+// testCheckResourceAttrLazy works like resource.TestCheckResourceAttr, but lazy evaluates the value parameter.
```

Comment:
Usually `resource.TestCheckResourceAttrPtr` can be used. Might be good to add that context here.

```suggestion
// testCheckResourceAttrLazy works like resource.TestCheckResourceAttr, but lazy evaluates the value parameter.
// See also: resource.TestCheckResourceAttrPtr.
```

Reply:
I'll reference it. In the case we are using the *lazy* function here, we are calling a function on the ptr, so that wouldn't be solved with the *ptr* function, right?

Reply:
Yes in your case the function makes sense since you need to transform a value on the issue object.

Reply:
Looks like your force-push reverted the applied suggestions.